src/org/jetbrains/java/decompiler/main/ClassWriter.java

```diff
@@ -1336,6 +1242,108 @@ else if (methodWrapper.varproc.getVarFinal(new VarVersionPair(index, 0)) == VarT
     return !hideMethod;
   }
 
+  private static int getParamCount(StructMethod mt, TextBuffer buffer, int indent, MethodWrapper methodWrapper, MethodDescriptor md, boolean isEnum, boolean init, boolean thisVar, GenericMethodDescriptor descriptor, int paramCount, boolean isInterface, int flags, StructClass cl) {
```

Comment:
I think this could use a slightly more descriptive name, since it has side effects as well as returning the param count.